.github/workflows/publish-commit-bottles.yml

```
@@ -44,7 +44,7 @@ jobs:
             github.issues.createComment({
               ...context.repo,
               issue_number: pr,
-              body: '@' + actor + ' has triggered a merge, running bottle pull: ' + url
+              body: '@' + actor + ' has triggered a merge. [View the progress logs](%s).', url
```

At the risk of bikeshedding, how about:

```suggestion
              body: '@' + actor + ' has triggered a merge. [View its progress](%s).', url
```

---

```suggestion
              body: '@' + actor + ' has triggered a merge. [View the merge logs](%s).', url
```

❓

---

```suggestion
              body: '@' + actor + ' has [triggered a merge](%s).', url
```

---

Yeah I was thinking about the last one. Generally, the less words the better.